js/client/modules/@arangodb/testsuites/recovery.js

```diff
@@ -175,7 +175,7 @@ function runArangodRecovery (params, useEncryption, isKillAfterSetup = true) {
   let agentArgs = pu.makeArgs.arangosh(params.options);
   agentArgs['server.endpoint'] = params.instanceManager.findEndpoint();
   if (params.args['log.level']) {
-    agentArgs['log.level'] = params.args['log.level'];
+    agentArgs['log.level'] = ['warning', 'httpclient=debug', 'V8=debug'];
```

Comment:
It is unclear if this change was made intentionally, or if it is a leftover from some debugging attempt?
If it is intentional, it is fine. The motivation is just unclear from looking at the change.

Reply:
I had already reverted that, but hadn't pushed it yet to avoid cancelling the running build. Pushed it just now,